meta-balena-zeus/recipes-core/initrdscripts/files/console_null_workaround

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
```

Comment:
Can this be moves to meta-balena-common?

Reply:
Can be. But I want to force myself to keep it outside so that I try to upstream it. :)